internal/asm/f64/asm_test.go

```diff
@@ -10,6 +10,13 @@ import (
 	"testing"
 )
 
+const (
+	msgVal      = "%v: unexpected value at %v Got: %v Expected: %v"
+	msgGuard    = "%v: Guard violated in %s vector %v %v"
+	msgReadOnly = "%v: modified read-only %v argument"
+	epsilon     = 1e-8
```

Comment:
I think for f64 blas functions this tolerance can/should be tighter. Except for the solve functions there is no source of inaccuracy.